### ISSUE TITLE
Fill the screen on mobile

### ISSUE THREAD

**Issue by teolemon** [openfoodfacts/hunger-games#478]
### Screenshots
![Screenshot_20230114-142518.png](https://user-images.githubusercontent.com/1689815/212474364-91389f54-ae3b-4488-8b0b-444cf0c93e2b.png)

### Part of
- (Insert Story issue number here)

**Comment by Sudhanva-Nadiger** [comment]
Can I work on this issue ?

**Comment by alexfauquette** [comment]
If you want, but #427 is a bit more interesting. Logo annotation might never be a good experience on mobile 🤷

**Comment by teolemon** [comment]
@alexfauquette you should remove your 👍 to stop it from adding top issue

**Comment by Sudhanva-Nadiger** [comment]
Yeah ! First i will work on regex search implementation! Then I will see this 👍🏻😁
